src/pages/edge/adobe-journey-optimizer-decisioning/assurance-setup.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Assurance setup
+description: This document shows user how to setup Assurance for the document features to be used.
+keywords:
+- Assurance setup
+---
+
+# Assurance setup
+
+This document shows user how to setup Assurance for the document features to be used.
+
+## Setup
+
+1. To get started you will need Adobe Experience Platform Assurance Extension to be enabled and set up in the application. Follow the instruction provided in [**Assurance Overview**](../../home/base/assurance/index.md) for setup.
+
+   If enabled, you would see Assurance Segment in the left panel of Data Collection App.
```

Comment:
The document you've linked in the previous line provides instructions on including Assurance SDK in the app. It is not related to enabling Assurance on Data Collection UI. You change it to something like this

"Next, navigate to Assurance UI and create and connect to a new session. You can find the instructions for it here https://experienceleague.adobe.com/en/docs/experience-platform/assurance/tutorials/using-assurance"